snapshot-labs/sx-monorepo
# feat: Labels settings for offchain spaces

Towards https://github.com/snapshot-labs/workflow/issues/143

### Summary: 
- Add new tab "labels" to off-chain space settings
- Admins should be able to add/edit/delete labels

### How to test:
- Go to labels settings on your off-chain spaces, ex: http://localhost:8080/#/s:thanku.eth/settings
- You should be able to see new tab labels (only in DEV mode for now)
- Should match design https://www.figma.com/design/hmHTau1Y1KYzrBBKLQr5QV/%23design?node-id=6906-14364&node-type=canvas&t=ui9jA40gPEdxm132-0
- Will open modals instead of new design https://discord.com/channels/955773041898573854/1287217523619528766/1290306249421881395
- You should be able to add new labels, edit or delete them

### TODO: 
- [ ] Merge only when we able to show labels on proposals and editor
